scripts/django2_migrate.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash -eux
+
+if [ $# -eq 0 ]; then
+    echo "No database provided"
+    exit 1
+fi
+
+# Check sqlite3 is installed
+sqlite3 --version
+
+# Delete invalid test accounts and create a dump for data incompatible with Django2 migrations
+cat ./scripts/django2_migration.sql | sqlite3 $1
+
+# Migrate the latest Django2 DB changes
+./manage.py migrate
+
+# Import previously dumped data after the migration
+sed -i '1i BEGIN TRANSACTION;' /tmp/assopy_invoice.sql
+echo 'END TRANSACTION;' >> /tmp/assopy_invoice.sql
```

Comment:
Why do these three models need to be migrated this way?

Reply:
Some FOREIGN KEY relations between the data in these 3 models (in the current ep2020 database) don't allow for proper run of the django2 migrations. 

Not sure entirely why, but it might be related to the change in sqlite3 that cause some issues with Django2 in the past, namely https://code.djangoproject.com/ticket/29182 as the error I am getting when running migrate after the django package update is this:

```
django.db.utils.IntegrityError: The row in table 'conference_vototalk' with primary key
'1' has an invalid foreign key: conference_vototalk.talk_id contains a value '318' that 
does not have a corresponding value in conference_talk__old.id.
```

For now, this is the only solution I could think of, as I don't want to go too deep to debug the data and loose too much time. 

Suggestions are welcome though, if someone has more experience with django and sqlite migrations.

Reply:
One possible "solution" is to wipe the db and all migrations, run ./manage.py makemigrations and then ./manage.py migrate...